src/scripts/db_setup.py

```diff
@@ -1,8 +1,13 @@
 from hack_path import hack_path
+import sqlite3
+from pathlib import Path
 
 hack_path()
 from db import db
 
+db_path = Path(__file__).parent.parent / "db.sqlite3"
+db = sqlite3.connect(db_path)
```

Comment:
This is really cool but `db` is already set up and in scope, it's imported the line before this 
that way you can configure the db path with the environment variable, and so that this script and `main.py` don't confusingly disagree about the DB filename 

this is the only comment I would genuinely change